modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/InstanceResourceTest.java

```diff
@@ -221,13 +225,58 @@ public void testGetProcessInstancesPageWithSortDateTime() throws Exception {
 			});
 	}
 
+	@Override
+	@Test
+	public void testGetProcessInstancesPageWithSortString() throws Exception {
+		testGetProcessInstancesPageWithSort(
+			EntityField.Type.STRING,
+			(entityField, instance1, instance2) -> {
+				String entityFieldName = entityField.getName();
+
+				if (StringUtil.equals("userName", entityFieldName)) {
+					User user1 = _addUser("aaa");
+
+					instance1.setCreator(
+						new Creator() {
+							{
+								id = user1.getUserId();
+								name = user1.getFullName();
+							}
+						});
+
+					User user2 = _addUser("bbb");
+
+					instance2.setCreator(
+						new Creator() {
+							{
+								id = user2.getUserId();
+								name = user2.getFullName();
+							}
+						});
+				}
```

Comment:
Hey @MarinhoFeliphe, can we use the other overloaded `setCreator` option? The code should look like the code below:

```java
instance1.setCreator(
  () -> {
     User user = _addUser("aaa");
    
     return new Creator() {
	    {
		    id = user.getUserId();
		    name = user.getFullName();
	    }
      };
  });
```

Thx